Frends.AzureBlobStorage.ListBlobsInContainer/.editorconfig

```diff
@@ -0,0 +1,4 @@
+﻿[*.cs]
+
+# CS1591: Missing XML comment for publicly visible type or member
+dotnet_diagnostic.CS1591.severity = suggestion
```

Comment:
This should not be in the repo and we would kind of need to do lots of comments. I would also advise to revert this to original severity locally :)

Reply:
Reverted to original. Thought I'd reverted this one. (Mistake from alt+enter)